tags/guide/fabricdebug.ytag

```diff
@@ -0,0 +1,13 @@
+type: text
```

Comment:
could probably do with an embed? Just something to consider. (This is not part of the requested changes, only a suggestion)